.rubocop_todo.yml

```diff
@@ -114,7 +114,7 @@ RSpec/ExampleLength:
 # Offense count: 2
 # Configuration parameters: Include, CustomTransform, IgnoreMethods, SpecSuffixOnly.
 # Include: **/*_spec*rb*, **/spec/**/*
-RSpec/FilePath:
+RSpec/SpecFilePathSuffix:
   Exclude:
     - 'spec/unit/puppet/provider/file_line/ruby_spec_alter.rb'
     - 'spec/unit/puppet/provider/file_line/ruby_spec_use_cases.rb'
```

Comment:
Please fix these instead by renaming the files. They are tests that don't get executed.